### DISCUSSION TITLE
Trigger @focus event

### DISCUSSION THREAD

**Discussion by frerotg** [vueform/vueform#293] [Ideas]
Hello,

I'm not able to trigger the basic event onfocus on TextElement directly with the event property like @change or @blur.
Am I missing something or is this deliberate?

Thanks for your response

**Comment by adamberecz** [comment]
It's currently not implemented, so I've moved it into the Ideas section.

**Comment by adamberecz** [comment]
Added [`@focus`](https://vueform.com/reference/text-element#event-focus) event in `1.10.2`
